docs/sources/enterprise/access-control/fine-grained-access-control-references.md

```diff
@@ -11,28 +11,32 @@ The reference information that follows complements conceptual information about
-| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Grafana Admin | `fixed:permissions:admin:edit`<br>`fixed:permissions:admin:read`<br>`fixed:provisioning:admin`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`<br>`fixed:users:admin:edit`<br>`fixed:users:admin:read`<br>`fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:ldap:admin:edit`<br>`fixed:ldap:admin:read`<br>`fixed:server:admin:read`<br>`fixed:settings:admin:read`<br>`fixed:settings:admin:edit` | Allow access to the same resources and permissions the [Grafana server administrator]({{< relref "../../permissions/_index.md#grafana-server-admin-role" >}}) has by default. |
-| Admin         | `fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`                                                                                                                                                                                                                                                                                                            | Allow access to the same resources and permissions that the [Grafana organization administrator]({{< relref "../../permissions/organization_roles.md" >}}) has by default.                 |
-| Editor        | `fixed:datasource:editor:read`                                                                                                                                                                                                                                                                                                                                                                                              |
+| Built-in role | Associated role                                                                                                                                                                                                                                                                                                                                                                                                             | Description                                                                                                                 |
```

Comment:
Modified descriptions and added two roles to admin `fixed:datasources:admin` `fixed:datasources:permissions:admin`
one role to Viewer: `fixed:datasources:id:viewer`